package-lock.json

```diff
@@ -110942,17 +110942,378 @@
         "@solana/web3.js": "1.93.4",
         "bn.js": "^5.2.1",
         "chalk": "^5.0.1",
-        "commander": "^9.4.0"
+        "commander": "^9.4.0",
+        "esbuild": "^0.24.0"
       },
       "bin": {
-        "audius-cmd": "src/index.mjs"
+        "audius-cmd": "dist/index.js"
       },
       "devDependencies": {
         "bats": "^1.8.2",
         "bats-assert": "^2.0.0",
         "bats-support": "0.3.0"
       }
     },
+    "packages/commands/node_modules/@esbuild/aix-ppc64": {
```

Comment:
oops I might redo this because I think I ran the install command inside the container the first time and it added all these arch dependencies.